applications/tari_base_node/src/grpc/base_node_grpc_server.rs

```diff
@@ -637,6 +646,43 @@ impl tari_rpc::base_node_server::BaseNode for BaseNodeGrpcServer {
         debug!(target: LOG_TARGET, "Sending GetTokensInCirculation response to client");
         Ok(Response::new(rx))
     }
+
+    async fn get_sync_info(
+        &self,
+        _request: Request<tari_rpc::Empty>,
+    ) -> Result<Response<tari_rpc::SyncInfoResponse>, Status>
+    {
+        debug!(target: LOG_TARGET, "Incoming GRPC request for BN sync data");
+
+        let mut channel = self.state_machine.get_status_info_watch();
+
+        let mut sync_info: Option<BlockSyncInfo> = None;
+
+        if let Some(info) = channel.recv().await {
```

Comment:
I think this needs to be: 
```suggestion
        if let Some(info) = channel.borrow().await {
```

Reply:
Says it is unimplemented.